src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -1,11 +1,72 @@
 package seedu.addressbook.commands;
 
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+/**
+ * Edits a person in the address book.
+ */
 public class EditCommand extends Command {
 
+	public static final String COMMAND_WORD = "edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" + "Edit a person in the address book. "
+            + "Contact details can be marked private by prepending 'p' to the prefix.\n\t"
+            + "Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...\n\t"
+            + "Example: " + COMMAND_WORD
+            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney";
```

Comment:
So, if I only want to change the phone number, I still have to repeat the person's name, email, address and tags? That's quite clunky, but I guess we can always improve the interface later.

What I'm more concerned is the fact that the specified name is used to search for the person to edit. This means that it is not possible to modify the name of an existing person.

Rather than finding the person to edit by its name, I wonder if the "edit" command could instead take an index of the person to edit?